Podcast/PastSearchTableViewCell.swift

```diff
@@ -7,40 +7,57 @@
 //
 
 import UIKit
+import SnapKit
+
 class PastSearchTableViewCell: UITableViewCell {
```

Comment:
Can we make this something like "PreviewResultSearchTableViewCell"

Reply:
how about PreviousSearchResultTableViewCell?